tests/playwright/sanity/modules/nested-tabs/nested-tabs-atomic-repeaters.test.ts

```diff
@@ -4,14 +4,13 @@ import WpAdminPage from '../../../pages/wp-admin-page';
 import { addItemFromRepeater, cloneItemFromRepeater, deleteItemFromRepeater, setup } from './helper';
 import _path from 'path';
 
-test.describe( 'Nested Tabs experiment is active @nested-atomic-repeaters', () => {
+test.describe( 'Nested Tabs experiment is active', () => {
```

Comment:
I would add back the suite, even if it's not experiment dependent, it's still nested atomic repeaters